pkg/context/context.go

```diff
@@ -24,7 +24,10 @@ type Context struct {
 	PathOverrides        option.PathOverrides
 	snapshotUnpackedPath string
 	alert                option.Alerter
-	err                  error
+	Err                  error
+	DisableNodeCaches    bool
+	DisableNodeAreas     bool
+	DisableNodeDistances bool
```

Comment:
do we need this (kinda unprecedented) level of detail or can we just have a single option controlling everything?